dockerfiles/Cli.dockerfile

```diff
@@ -43,7 +25,16 @@ COPY ./packages/utils/package.json /proj/packages/utils/package.json
 COPY ./packages/zk-wizard/package.json /proj/packages/zk-wizard/package.json
 COPY ./yarn.lock /proj/yarn.lock
 
-RUN yarn install
+
+RUN npm install -g lerna
+# install build tools
+RUN apk add --no-cache --virtual .gyp \
```

Comment:
Ideally we'd combine commands like this so that we have fewer docker layers. Not a huge concern though.

Reply:
I am going through the dockerfiles to apply your feedback. So, in general, consecutive RUN layers can be grouped or are there some cases where they are better as separate layers ?

Reply:
There are some cases, usually when you have something that doesn't always need to be done you will group it separately from things that _do_ always need to be done.

Like an install and a build are sometimes better as separate layers (though they're usually separated by a `COPY` step). Generally you'll know when things are better as separate layers. I've also run into weird cases where having one large `RUN` step produces a larger image than a few smaller `RUN` steps. It's a not a huge issue, just something that's good to know.

Reply:
thank you for the detailed answer